### DISCUSSION TITLE
prefix + mousePositionControl + suffix

### DISCUSSION THREAD

**Discussion by prospoule** [openlayers/openlayers#16179] [Q&A] [answered]
Hello, does anybody knows how to add a prefix and/or a suffix to a mousePositionControl in OL10 ?
So far all I found is the ability to add a placeholder and to change the fractional digit:
```
const format = createStringXY(1);
mousePositionControl.setCoordinateFormat(format);
```
Thank you

**Comment by ahocevar** [comment, marked as answer]
`coordinateFormat` is simply a function that receives a coordinate and returns the desired output. So you could do something like
```js
const toString = createStringXY(1);
const format = (coordinate) => `Prefix ${toString(coordinate)} Suffix`;
mousePositionControl.setCoordinateFormat(format);

**Comment by prospoule** [comment]
It perfectly worked. Thank you for taking the time to answer me.
